packages/http-server-express/package.json

```diff
@@ -21,13 +21,15 @@
 	"license": "MIT",
 	"dependencies": {
 		"@davinci/http-server": "^2.1.2",
-		"@davinci/reflector": "^2.1.1"
+		"@davinci/reflector": "^2.1.1",
+		"cors": "^2.8.5"
```

Comment:
please move this to peerDependencies
also copy it in the devDependencies.

Have a look at the @fastify/cors dependency for inspiration: https://github.com/HPInc/davinci/blob/master/packages/http-server-fastify/package.json